egs/mandarin_voice/s1/03_prepare_acoustic_features.sh

```diff
@@ -32,7 +32,7 @@ copy=true
 if [ "$prepare_feats" = true ]; then
     echo "Step 3:" 
     echo "Prepare acoustic features using "${Vocoder}" vocoder..."
-    python ${MerlinDir}/misc/scripts/vocoder/${Vocoder,,}/extract_features_for_merlin.py ${MerlinDir} ${wav_dir} ${feat_dir} $SamplingFreq 
```

Comment:
Hi, which vocoder are you using? I don't see why it fails

Reply:
shell variables is ${Vocoder}，not ${Vocoder,,}

Reply:
thx for that, my mistake